recipe/meta.yaml

```
@@ -1,52 +1,100 @@
 {% set name = "smac" %}
-{% set version = "2.1.0" %}
-
+{% set version = "1.2" %}
+{% set git_version = "1.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: af9d3bffd4d8d233ecc872a8c0f7c245043d41340a7790091165d363b2f7fe63
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: f98e2c6baa91dd275ca4f0fedb47a0d38fe5875564a0805b6b1c6b8866561c24
+    patches:
+      - patches/0001-fix-numpy-deprecated-float-to-float64.patch
+      - patches/0002-fix-numpy-deprecated-int-to-int64.patch
+      - patches/0003-n_restarts_optimizer-must-be-positive.patch
+  # use GH release for missing data files
+  - url: https://github.com/automl/SMAC3/archive/refs/tags/v{{ git_version }}.tar.gz
+    sha256: 72d8874ded51abeb0a775dc7056548b0c4a5d82ea74edbb066ddde633437a5ca
+    patches:
+      # same patch here because a test file is patched as well
+      - patches/0002-fix-numpy-deprecated-int-to-int64.patch
+    folder: gh_src
 
 build:
   number: 0
-  noarch: python
+  # skip win because pynisher 0.6.4 unavailable
+  skip: True  # [py<38 or win]
```

```suggestion
  skip: True  # [py<37 or win]
```
https://github.com/automl/SMAC3/blob/v1.2.0/setup.py#L37

---

ok, thanks, nice catch